src/SourceBuild/content/eng/pipelines/ci-lite.yml

```diff
@@ -0,0 +1,34 @@
+# This is the main build definition (PR+CI) for dotnet/dotnet-lite
```

Comment:
This wording is confusing because at first I thought dotnet/dotnet-lite was referring to a repo.  Additionally it feels strange that both pipelines refer to themselves as the main pipeline now that there are two pipelines.  Lastly I personally try to avoid having the yml reference the pipeline name it is referenced by because it is easy for them to get out of sync.

I prefer removing the first line from both pipelines and reword the second along the lines of 

```
This is the dotnet/dotnet pipeline that is triggered...
```

Reply:
Updated in https://github.com/dotnet/installer/pull/17370/commits/a0f1e49155c565fc1052096bdbeec7b0ccd9873b